src/main/java/com/google/jenkins/plugins/storage/util/StorageUtil.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jenkins.plugins.storage.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+import com.google.jenkins.plugins.storage.Messages;
+import com.google.jenkins.plugins.storage.UploadException;
+import hudson.FilePath;
+import hudson.Util;
+import hudson.model.AbstractBuild;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import java.io.IOException;
+import java.util.LinkedList;
+
+/**
+ * A class to contain common utility operations
+ */
+public class StorageUtil {
+  /**
+   * Prefix the given log message with our module.
+   */
+  public static String prefix(String moduleName, String x) {
```

Comment:
Can this be removed then (and the name of the PrefixFormat message reverted if it is only referenced from UploadModule)?

Reply:
Or, alternatively, make UploadModule.prefix call StorageUtil.prefix if this will be used later from elsewhere.

Reply:
Thanks! I was certain I removed the prefix in StorageUtil in my last update.
Must have forgotten to "git add" :-(

It's there now.